src/sentry/migrations/0704_backfill_rule_user_team.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 5.0.3 on 2024-04-23 14:55
+
+import click
+from django.apps.registry import Apps
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.models import Q
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_rule_owners(apps: Apps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    Rule = apps.get_model("sentry", "Rule")
+    Actor = apps.get_model("sentry", "Actor")
+
+    expr = Q(owner_user_id__isnull=True) | Q(owner_team_id__isnull=True)
+
+    rules = Rule.objects.filter(owner_id__isnull=False).filter(expr)
+    for rule in RangeQuerySetWrapperWithProgressBar(rules):
+        actor = Actor.objects.get(id=rule.owner_id)
+        changed = False
+        if actor.user_id:
+            rule.owner_user_id = actor.user_id
+            changed = True
+        elif actor.team_id:
+            rule.owner_team_id = actor.team_id
+            changed = True
+        else:
+            click.echo(f"Actor {actor.id} is neither a user or team")
```

Comment:
Is using click ok in a migration?

Reply:
I hope so, because print() is banned by our flake8 rules.

Reply:
Should we use logging instead?